src/codewhisperer/util/authUtil.ts

```diff
@@ -104,6 +104,9 @@ export class AuthUtil {
                     vscode.commands.executeCommand('aws.codeWhisperer.gettingStarted')
                     prompts.disablePrompt('codeWhispererNewWelcomeMessage')
                 }
+
+                // start the feature config polling job
+                vscode.commands.executeCommand('aws.codewhisperer.fetchFeatureConfigs')
```

Comment:
if this is should be executed on IDE start, are we able to move it to `activation.ts`?

Reply:
I thought about this but meanwhile wee need to get CodeWhisperer logged in first so we can invoke the api. If we put it in `activation.ts`, CodeWhisperer may not be logged in at that time and we will miss reading this. That's why I chose to do it every time after we get a valid connection.

Reply:
sgtm